optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/micro/AbstractPlannerMoveMicroBenchmark.java

```diff
@@ -0,0 +1,51 @@
+package org.jboss.qa.brms.performance.micro;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.optaplanner.core.impl.heuristic.move.Move;
+import org.optaplanner.core.impl.score.director.ScoreDirector;
+
+@State(Scope.Thread)
```

Comment:
You are missing warmup and measurement annotations here which define the number of iterations. Please don't miss properly checking, if the JVM is warmed up correctly when setting these values.

Reply:
Added annotations. Thanks!